library/entropy_poll.c

```diff
@@ -5,7 +5,7 @@
  *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
  */
 
-#if defined(__linux__) && !defined(_GNU_SOURCE)
+#if defined(__linux__) || defined(__midipix__) && !defined(_GNU_SOURCE)
```

Comment:
Note: missing `(` and `)` around the `||` sub-expression here; this should have been

```
#if (defined(__linux__) || defined(__midipix__)) && !defined(_GNU_SOURCE)
```

Reply:
(now being fixed in #9026)